src/contracts/core/StrategyManagerStorage.sol

```diff
@@ -26,6 +26,9 @@ abstract contract StrategyManagerStorage is IStrategyManager {
     // index for flag that pauses deposits when set
     uint8 internal constant PAUSED_DEPOSITS = 0;
 
+    /// @notice default address for burning slashed shares and transferring underlying tokens
+    address public constant DEFAULT_BURN_ADDRESS = address(0);
```

Comment:
can we do some analysis on if this is ok for LSTs?

Reply:
It's for sure not, placeholder value for now. Personally a fan of 0xdeadbeef

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/881/commits/91a806b48e0c665ef6389749b6582b3c7ec01c18 
0xDeaDBeef

Reply:
We should at least do `0x000000000000000000000000000000000000dEaD` or address(0) since other protocols use that. Working with Matt Nelson to get feedback

Reply:
Looks like we cannot do address(0) given that stETH's contract